### PR TITLE
Fix tracking response parsing for shipment events without location

### PR DESCRIPTION
**Pull request by vitaliel**

### CODE REVIEW

**Comment by vitaliel** [comment]
@barhot @ajlai @Aryk please review

File: lib/active_shipping/shipping/carriers/ups.rb [hunk 1]
```diff
@@ -45,15 +45,15 @@ class UPS < Carrier
           :occasional
         end
       end
-      
+
       DEFAULT_SERVICES = {
         "01" => "UPS Next Day Air",
-        "02" => "UPS 2nd Day Air",
+        "02" => "UPS Second Day Air",
```
Comment (denis) [inline lib/active_shipping/shipping/carriers/ups.rb]: What is the reason behind this change? `UPS 2nd Day Air` is the actual name: https://www.ups.com/content/us/en/shipping/time/service/second_day.html

File: lib/active_shipping/shipping/carriers/ups.rb [hunk 2]
```diff
@@ -45,15 +45,15 @@ class UPS < Carrier
...
         "03" => "UPS Ground",
         "07" => "UPS Worldwide Express",
         "08" => "UPS Worldwide Expedited",
         "11" => "UPS Standard",
-        "12" => "UPS 3 Day Select",
+        "12" => "UPS Three-Day Select",
```
Comment (denis) [inline lib/active_shipping/shipping/carriers/ups.rb]: The same here: https://www.ups.com/content/us/en/shipping/time/service/three_day.html?WT.mc_id=VAN701413

File: lib/active_shipping/shipping/carriers/ups.rb (hunk 1 above)
Comment (vitaliel) [inline lib/active_shipping/shipping/carriers/ups.rb]: @denis I made the change to make tests pass. I will revert and fix tests.

**Comment by vitaliel** [comment]
@denis please review

**Comment by Aryk** [comment]
LGTM

**Comment by ajlai** [comment]
LGTM

**Comment by barhot** [comment]
LGTM
